app/Response/Draw.hs

```diff
@@ -37,15 +38,17 @@ modePanel = H.div ! A.id "side-panel-wrap" $ do
             ! A.type_ "text"
             ! A.size "10"
     H.div ! A.id "add-text" ! A.class_ "button" $ "ADD"
-    H.div ! A.id "red" ! A.class_ "colour clicked" $ "RED"
-    H.div ! A.id "green" ! A.class_ "colour" $ "GREEN"
-    H.div ! A.id "blue" ! A.class_ "colour" $ "BLUE"
-    H.div ! A.id "purple" ! A.class_ "colour" $ "PURPLE"
     H.div ! A.id "path-mode" ! A.class_ "mode" $ "PATH (p)"
     H.div ! A.id "region-mode" ! A.class_ "mode" $ "REGION (r)"
     H.div ! A.id "finish-region" ! A.class_ "button" $ "finish (f)"
     H.div ! A.id "change-mode" ! A.class_ "mode" $ "SELECT/MOVE (m)"
     H.div ! A.id "erase-mode" ! A.class_ "mode" $ "ERASE (e)"
+    H.input ! A.id "select-colour"
+            ! A.class_ "jscolor"
+            ! A.value "ab2567"
+            ! A.size "15"
+    H.table !A.id "colour-table" $ forM_ ([replicate 5 "", replicate 5 ""] :: [[String]])
```

Comment:
Hmm string literals can be interpreted as `Html` type, so I wonder if you can do `[[Html]]` so that you don't need to call `H.toHtml` below